src/main/java/de/adesso/anki/battle/GameEngine.java

```diff
@@ -49,6 +53,38 @@ public void start() {
 
     	 
     }
+    //TODO set ready at the same time, 

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)